### DISCUSSION TITLE
IDH Node

### DISCUSSION THREAD

**Discussion by ColeVan** [Security-Onion-Solutions/securityonion#7522] [Unsupported Versions] [answered]
Question? Can IDH node be joined to a standalone setup or does it have to be a distributed deployment architecture?

**Comment by RagnarSTS** [comment]
Conversion is definitely not supported.  I don't think it is pretty to try and convert a standalone to a distributed.   I just rebuilt my home lab setup for the same reason.  For home lab I just ran standalone, but training  lab was using distributed.  


Some notes from my recent redeployment for distributed:
- Sensors load up much faster; during reboot cycles they'll come up first and simply hold data until the mastersearch is up. Big benefit over standalone setup
- Install supports 2 drives; one for / and one for nsm.  nsm was platter drive due to pcap.  Then required hackery to move the elasticsearch folders off onto a third separate ssd to make things more responsive. Separate Master/Search and Sensor nodes make this division much easier.  Master/Search on SSD.  Sensor on SSD with nsm on platter.  Perfect. Big benefit over standalone.
- If you are resource restricted in a small environment; was able to reduce RAM required a bit on master.  Almost made up entirely for the extra vm added.
- Honeypot node is definitely one you want to thin provision on the hard drive; 100G is overkill even considering the ridiculous overhead that containers eat while updating. Total basic install is less than 20G; and probably won't ever get over 40G.
- Compared to a barebone debian machine running a honeypot that simply sends email alerts; this uses approximately 3x more CPU. Seems to idle around 0.4% CPU and frequently bounces up to 4 or 5% utilization.  Other machine on same hardware is less than 0.2% usage for most of the day.
- I think it is worth the extra resources to have it nicely integrated into the SIEM properly.

**Reply by b4b857f6ee** [reply]
Yes, if we can have a small 20Go appliance, this can be huge!

**Comment by defensivedepth** [comment, marked as answer]
Yes, an IDH node can be joined to a Standalone - you will just need to remember what you set the `so-remote` user password to when you initially setup your Standalone :)

@RagnarSTS We are still benchmarking the minimum requirements for an IDH node. I believe most of the CPU/Mem usage during idle is from the Wazuh container, which is actually optional.
